docs/articles/oso/checklist-for-making-software-open-source.rst

```diff
@@ -0,0 +1,38 @@
+:orphan:
+
+Checklist for making software open source
+=========================================
+
+.. tagslist:: howto
+
+When making software open source there are some steps that need to be completed.
+The purpose of these steps is to ensure that the code has a sound license, the
+source code repository is understandable and that there are clear descriptions
+on how to contribute. Following is a listing of these steps:
+
+#. Pick an open source license [#owning_copyright_required]_.
+#. Make sure that the license of all dependencies of the software fit the
+   desired license.
+#. Make sure that the following files are present in the root of the source code
+   repository:
+    * LICENSE: Containing the desired license for the code
+    * README.md: Containing information on where to find the documentation and
+      simple getting started info.
+    * CONTRIBUTING.md: Containing information on how to contribute to the
+      project and what rules there are.
+#. Go through file headers of all the source code and make sure that there is a
+   copyright notice, a reference to the LICENSE file and a SPDX tag
+   [#owning_copyright_required]_.
+#. Send a request to make the software open source to the Open Source Office
```

Comment:
This assumes there is an OSO available. In that case we should describe what that is, either here, or somewhere else and link to it.

Reply:
It should not be described in a how-to like this. I guess we could add a placeholder article with a brief description for now if we feel that's needed. I agree it is slightly strange to refer to something that doesn't exist yet, but it's also pretty difficult to get away from in these situations where we are adding content from many directions. I know @jeremiah will add content about OSO soon though. If we just have a small article added now, we can change that later.